pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -931,27 +959,11 @@ def create_model(df_sets, df_parameters, default={}):
         initialize={},
         doc="Valid storage-to-reuse pipeline arcs [-]",
     )
-    df_parameters["LLP"] = {
-        **df_parameters["PNA"],
-        **df_parameters["CNA"],
-        **df_parameters["CCA"],
-        **df_parameters["NNA"],
-        **df_parameters["NCA"],
-        **df_parameters["NKA"],
-        **df_parameters["NSA"],
-        **df_parameters["NRA"],
-        **df_parameters["FCA"],
-        **df_parameters["RCA"],
-        **df_parameters["RNA"],
-        **df_parameters["SNA"],
-        **df_parameters["RSA"],
-        **df_parameters["SCA"],
-    }
     model.p_LLP = Param(
```

Comment:
```suggestion
    model.p_LLA = Param(
```

Reply:
I like this. Turns out this parameter isn't used anywhere in the model, so I'm removing it.